player-service/src/main/java/org/libertybikes/player/service/PlayerService.java

```diff
@@ -1,53 +1,88 @@
 package org.libertybikes.player.service;
 
 import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.UUID;
+import java.util.Random;
 
 import javax.annotation.PostConstruct;
 import javax.enterprise.context.ApplicationScoped;
-import javax.ws.rs.Consumes;
+import javax.inject.Inject;
 import javax.ws.rs.GET;
+import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
 
-@Path("/")
+import org.libertybikes.player.data.PlayerDB;
+
+@Path("/player")
 @ApplicationScoped
 public class PlayerService {
 
-    Map<String, Player> allPlayers = new HashMap<>();
+    @Inject
+    PlayerDB db;
 
     @PostConstruct
     public void initPlayers() {
-        for (int i = 0; i < 10; i++)
-            createPlayer();
+        Random r = new Random();
+        for (int i = 0; i < 10; i++) {
+            String id = createPlayer("Bob-" + i);
```

Comment:
Was this meant to be checked in? Is this for a test, or just temporary until we implement something else?

Reply:
oops, I added a TODO comment locally but forgot to push it up.  Yes the idea here is that we can easily seed some example data without needing to create it manually each time.